swagger.yaml

```diff
@@ -6163,26 +6217,15 @@ definitions:
     type: string
     pattern: '^(?:(?:(?:[0-9]{1,3}\.){3}[0-9]{1,3}\/(?:(?:[0-9])|(?:[1-2][0-9])|(?:3[0-2])))|(?:(?:[0-9a-fA-F]*:[0-9a-fA-F]*){2,})/(?:(?:[0-9])|(?:[1-9][0-9])|(?:1[0-1][0-9])|(?:12[0-8])))$'
 
-  network_configuration:
-    type: object
-    properties:
-      cluster_network:
-        type: array
-        items:
-          $ref: '#/definitions/cluster_network'
-      machine_network:
-        type: array
-        items:
-          $ref: '#/definitions/machine_network'
-      service_network:
-        type: array
-        items:
-          $ref: '#/definitions/service_network'
-
   cluster_network:
     type: object
     description: IP address block for pod IP blocks.
     properties:
+      cluster_id:
+        type: string
+        format: uuid
+        description: The cluster that this network is associated with.
```

Comment:
If there is a primary key only for cluster-id, then there will be only a single record per cluster in this table - which is not what we want.